sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -86,6 +89,7 @@ object ScalaNativePluginInternal {
       .withGC(Discover.GC())
       .withMode(Discover.mode())
       .withOptimize(Discover.optimize()),
+    optimizerConfig := build.OptimizerConfig.empty,
```

Comment:
I don't think this line is needed at all since `OptimizerConfig.empty` is created when `NativeConfig.empty` is created.